src/components/util/BadComponents.tsx

```diff
@@ -0,0 +1,58 @@
+// BadComponent.tsx
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const Div = styled.div`
+  background-color: ${(props: any) => props.bg || 'white'};
+  color: ${(props: any) => props.color || 'black'};
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+
+const Btn = styled.button`
+  background-color: ${(props: any) => props.bg || 'blue'};
+  color: ${(props: any) => props.color || 'white'};
+  border: none;
+  padding: 10px;
+  margin-top: 10px;
+  cursor: pointer;
+  &:hover {
+    opacity: 0.8;
+  }
+`;
+
+const Txt = styled.p`
+  font-size: 1.5rem;
+`;
+
+const BadComponent: React.FC = () => {
+  const [count, setCount] = useState<any>(0);
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/taulant-eulerity/Canva/pull/4#issuecomment-2181154623>#57299f</a><span> - 06/20/2024, 05:05 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The use of 'any' type for state variables ('count', 'txt', 'bg') can lead to type safety issues and makes the code harder to maintain and understand.
</div>
<div id="fix">
<b>Fix:</b> Replace the 'any' type with more specific types for each state variable to ensure type safety and improve code readability.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -34,9 +34,9 @@ const BadComponent: React.FC = () => {
  const [count, setCount] = useState<number>(0);
  const [txt, setTxt] = useState<string>('Hello World');
  const [bg, setBg] = useState<string>('lightgray');
```
</div>
</details>
</div>
<br>


</div>